File: java/soma_matrizes.java
import java.util.Locale;
import java.util.Scanner;

public class soma_matrizes {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int m, n;

		System.out.print("Quantas linhas vai ter cada matriz? ");
	    m = sc.nextInt();
	    System.out.print("Quantas colunas vai ter cada matriz? ");
	    n = sc.nextInt();

	    int[][] a = new int[m][n];
	    int[][] b = new int[m][n];
	    int[][] c = new int[m][n];

	    System.out.println("Digite os valores da matriz A:");

	    for (int i=0; i<m; i++) {
	        for (int j=0; j<n; j++) {
	        	System.out.printf("Elemento [%d,%d]: ", i, j);
	            a[i][j] = sc.nextInt();
	        }
	    }

	    System.out.println("Digite os valores da matriz B:");

	    for (int i=0; i<m; i++) {
	        for (int j=0; j<n; j++) {
	        	System.out.printf("Elemento [%d,%d]: ", i, j);
	            b[i][j] = sc.nextInt();
	        }
	    }

	    for (int i=0; i<m; i++) {
	        for (int j=0; j<n; j++) {
	            c[i][j] = a[i][j] + b[i][j];
	        }
	    }

	    System.out.println("MATRIZ SOMA:");

	    for (int i=0; i<m; i++) {
	        for (int j=0; j<n; j++) {
	        	System.out.printf("%d ", c[i][j]);
	        }
	        System.out.println();
	    }

		sc.close();
	}
}

File: python/soma_matrizes.py
m: int; n: int

m = int(input("Quantas linhas vai ter cada matriz? "))
n = int(input("Quantas colunas vai ter cada matriz? "))

a: [[int]] = [[0 for x in range(n)] for x in range(m)]
b: [[int]] = [[0 for x in range(n)] for x in range(m)]
c: [[int]] = [[0 for x in range(n)] for x in range(m)]

print("Digite os valores da matriz A:")

for i in range(m):
	for j in range(n):
		a[i][j] = int(input(f"Elemento [{i},{j}]: "))

print("Digite os valores da matriz B:")

for i in range(m):
	for j in range(n):
		b[i][j] = int(input(f"Elemento [{i},{j}]: "))

for i in range(m):
	for j in range(n):
		c[i][j] = a[i][j] + b[i][j]

print("MATRIZ SOMA:")

for i in range(m):
	for j in range(n):
		print(f"{c[i][j]} ", end="")
	print()